tavern/tomes/file_mkdir/main.eldritch

```diff
@@ -0,0 +1,19 @@
+
+def main():
+    new_dir = input_params['path']
+    if file.exists(new_dir):
+        eprint("File/Directory already exists.")
+        return
+    parent = True
```

Comment:
Could we shorten this to:
```python
parent = input_params['parent'].lower() == "true"
```

Reply:
we can use `.lower` but we should error if they put something in like  `"blorp"`

Reply:
I think it's okay to default out instead of failing.